_commands/basics/su.md

```diff
@@ -0,0 +1,70 @@
+---
+---
+
+su
+---
+
+`su` (substitute user) is a command to change a login session's owner
+without the owner having to first log out of the session. (as specified 
+in the [su Command](https://www.techonthenet.com/linux/commands/su.php)). In
+other words, it is called the switch user command
+
+__Note:__ It is most commonly employed to change the ownership from
+an ordinary user to the administrative user. Therefore it is oftern
+refered as the superuser command.
+
+
+
+~~~ bash
+$ su root
```

Comment:
This won't work unless root has a password set. You'd have to `sudo su root`, which is exactly the same as `sudo su`, which I use most often.